### PR TITLE
ASPNET Core SignalR Supported platforms

### LINKED ISSUE #6015: ASP.NET Core SignalR: Supported Platforms Doc
We should get a doc out with the supported platforms for ASP.NET Core SignalR. This should be a fairly high priority for SignalR, as we've had some confusion around it in the previews.

The existing doc is https://docs.microsoft.com/en-us/aspnet/signalr/overview/getting-started/supported-platforms . I think the doc for Core can be a lot simpler, and doesn't need to dive in to which transports are supported where.

## Server system requirements

* This is much easier than the existing doc. We support any server platform ASP.NET Core supports.

## Client system requirements

Browser support:

* Microsoft Internet Explorer 11
* Microsoft Edge: current version
* Mozilla Firefox: current version
* Google Chrome: current version; includes Android.
* Safari: current version; includes iOS

.NET Client support:

* Any .NET platform that supports .NET Standard 2.0.

### PR DESCRIPTION
**Pull request by rachelappel**
Fixes #6015
[Review URL](https://review.docs.microsoft.com/en-us/aspnet/core/signalr/supported-platforms?view=aspnetcore-1.0&branch=pr-en-us-6021)

### CODE REVIEW

**Comment by rachelappel** [comment]
@anurse @BrennanConroy @mikaelm12 Adding you all so you're aware it's here. I realize it's a busy week.

File: aspnetcore/signalr/supported-platforms.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: ASP.NET Core SignalR supported platforms
+author: rachelappel
+description: Supported platforms for ASP.NET Core SignalR 
+manager: wpickett
```
Comment (scottaddie) [inline aspnetcore/signalr/supported-platforms.md]: Add the following metadata under the `manager` metadata:

`monikerRange: '>= aspnetcore-2.1'`

This causes the doc to display only when 2.1 (or later) is selected from the version selector.

File: aspnetcore/signalr/supported-platforms.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/supported-platforms
+---
+
+# ASP.NET Core SignalR supported platforms
+
+## Server system requirements
+
+SignalR for ASP.NET Core supports any server platform ASP.NET Core supports.
```
Comment (anurse) [inline aspnetcore/signalr/supported-platforms.md]: ...any server platform **supported by** ASP.NET Core ~~supports~~. **When using IIS, the WebSockets transport requires IIS 8.0 or higher, on Windows Server 2012 or higher. Other transports are supported on all platforms.**

File: aspnetcore/signalr/supported-platforms.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+### Browser support
+
+The SignalR for ASP.NET Core JavaScript client supports the following browsers:
+
+| Browser | Version |
+| ------- | ------- |
+| Microsoft Internet Explorer | 11 |
+| Microsoft Edge | current |
+| Mozilla Firefox | current |
+| Google Chrome | current; includes Android |
```
Comment (anurse) [inline aspnetcore/signalr/supported-platforms.md]: Let's put the "includes Android" and "includes iOS" in the Browser column actually, so "Google Chrome (including Android)" and "Safari (including iOS)"

**Pull request merged**